js/private/test/node-patches/opendir.mjs

```diff
@@ -52,7 +52,15 @@ describe('testing opendir', async () => {
                     path.join(fixturesDir, 'a')
                 )
                 const entry1 = await dir.read()
-                const entry2 = await util.promisify(dir.read.bind(dir))()
```

Comment:
This was causing a warning to stdout since `dir.read()` is already a promise-returning method, while `dir.read(callback)` is not.